bottlecap/src/bin/bottlecap/main.rs

```diff
@@ -344,7 +338,7 @@ async fn extension_loop_active(
                 invoked_function_arn,
             }) => {
                 debug!(
-                    "[extension_next] Invoke event {}; deadline: {}, invoked_function_arn: {}",
+                    "Invoke event {}; deadline: {}, invoked_function_arn: {}",
```

Comment:
Why is `extension_next` only here?

Reply:
i think AJ added it in a previous PR, but I just fixed it in the formatter